core/src/main/java/org/elasticsearch/action/fieldstats/FieldStats.java

```diff
@@ -550,27 +625,50 @@ public static FieldStats readFrom(StreamInput in) throws IOException {
         long sumTotalTermFreq = in.readLong();
         boolean isSearchable = in.readBoolean();
         boolean isAggregatable = in.readBoolean();
-
+        boolean hasMinMax = true;
```

Comment:
It is weird to see this pre-NamedWriteable stuff still around. It might make sense to convert these to NamedWriteable in master.